airbyte-json-validation/src/main/java/io/airbyte/validation/json/JsonSchemaValidator.java

```diff
@@ -58,7 +58,7 @@ public JsonSchemaValidator() {
    * @param baseUri The base URI for schema resolution
    */
   @VisibleForTesting
```

Comment:
nit: `@VisibleForTesting` can be removed. Also if class has the same package structure with the unit test all protected method is accessible for unit test

Reply:
unfortunately I'm using this in a separate test case from a different package (probably not a best practice, but it simplified writing the test cases) https://github.com/airbytehq/airbyte/pull/19909/files#diff-6829fbf8db73883ede3e47d869e6f5756a882c2d94d74c13e0b2128b07666216R36

Reply:
With the constructor being public now, I don't think we need the annotation anymore.

Reply:
I'd mildly prefer to keep it since it's still only being used by tests? I.e. production code paths should just use the no-args constructor